rosidl_typesupport_opendds_c/CMakeLists.txt

```diff
@@ -1,31 +0,0 @@
-cmake_minimum_required(VERSION 3.5)
```

Comment:
@jwillemsen There are a number of files that were removed here and most likely added to a new repository.  How do you want to handle that external repository?  Should we transfer it?

Reply:
I can tranfer it, just let me know which repo and to which organization, can do that tomorrow

Reply:
Looks like we should transfer https://github.com/jwillemsen/rosidl_typesupport_opendds to mitza-oci first (since it won't let you transfer to oci-labs), then I will transfer to oci-labs.

Reply:
I have made a transfer request for the repo to @mitza-oci

Reply:
Repo looks to be transferred